pkgs/by-name/rg/rgl/package.nix

```diff
@@ -0,0 +1,43 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  nix-update-script,
+  versionCheckHook,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "rgl";
+  version = "0.17.0";
+
+  src = fetchFromGitHub {
+    owner = "ink0rr";
+    repo = "rgl";
+    tag = "v${version}";
+    hash = "sha256-kucD1FosCV2dxhLzxpOv/HvsqddH3NiAEVkqGTWTmGE=";
+  };
+
+  cargoHash = "sha256-/BeT18tIucgq1b5js+7QX0d1OyF5drNWMAFtv+g/XtM=";
+
+  nativeInstallCheckInputs = [ versionCheckHook ];
+  doInstallCheck = true;
+
+  versionCheckProgram = "${placeholder "out"}/bin/${meta.mainProgram}";
```

Comment:
This line is redundant as it's the default behavior, please remove it.

Reference: https://nixos.org/manual/nixpkgs/unstable/#versioncheckhook